src/themes/default/components/core/blocks/Header/Header.vue

```diff
@@ -108,6 +111,9 @@ export default {
         .wishlist-icon {
```

Comment:
for all this icons add one proper class to visibility.scss

Reply:
I added hidden-xs class to icons